### ISSUE TITLE
string-interner 0.6 has been unpublished

### ISSUE THREAD

**Issue by jyn514** [m4b/faerie#93]
As of this morning, faerie doesn't compile with a clean cargo cache (see my [travis build log](https://travis-ci.org/jyn514/rcc/jobs/579502222), where I first saw this). Here's the message (reproducable with `cargo update`):

```
error: failed to select a version for the requirement `string-interner = "^0.6"`
  candidate versions found which didn't match: 0.7.1, 0.3.3, 0.3.2, ...
  location searched: crates.io index
required by package `faerie v0.10.2 (/home/joshua/Documents/Programming/rust/faerie)`
```

Looking at https://crates.io/crates/string-interner, it's because of a use-after-free bug that was fixed 4 hours ago. All other versions have been yanked from crates.io.

**Comment by jyn514** [comment]
I made a start on fixing this, but `string_interner::Sym` (which replaced `usize`) doesn't implement Borrow, so it can't be used in an indexmap. I don't want to make non-trivial changes without asking.

**Comment by jyn514** [comment]
Found more info on the advisory:

- https://github.com/RustSec/advisory-db/pull/138
- https://github.com/Robbepop/string-interner/issues/9#issuecomment-526857810

**Comment by pchickey** [comment]
Fixed in #94, part of 0.11.0 release
